.ci/setup_env_debian.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+arch=$("${cidir}"/kata-arch.sh -d)
+export DEBIAN_FRONTEND=noninteractive
+
+echo "Install chronic"
+sudo -E apt install -y moreutils
+
+echo "Install curl"
+sudo -E apt install -y curl
+
+echo "Install git"
+sudo -E apt install -y git
+
+echo "Install kata containers dependencies"
+chronic sudo -E apt install -y libtool automake autotools-dev autoconf bc alien libpixman-1-dev coreutils
+
+echo "Install qemu dependencies"
+chronic sudo -E apt install -y libcap-dev libattr1-dev libcap-ng-dev librbd-dev
+
+echo "Install kernel dependencies"
+chronic sudo -E apt install -y libelf-dev
+
+echo "Install CRI-O dependencies for Debian"
+chronic sudo -E apt install -y libglib2.0-dev libseccomp-dev libapparmor-dev \
+        libgpgme11-dev go-md2man thin-provisioning-tools
+
+echo "Install bison binary"
+chronic sudo -E apt install -y bison
+
+echo "Install libudev-dev"
+chronic sudo -E apt-get install -y libudev-dev
+
+echo "Install Build Tools"
+chronic sudo -E apt install -y build-essential python pkg-config zlib1g-dev
+
+echo -e "Install CRI-O dependencies available for Debian"
+chronic sudo -E apt install -y libdevmapper-dev btrfs-tools util-linux
+
+chronic sudo -E apt install -y libostree-dev
+
+echo "Install YAML validator"
+chronic sudo -E apt install -y yamllint
+
+echo "Install tools for metrics tests"
+chronic sudo -E apt install -y smem jq
+
+echo "Enable librbd1 repository"
+sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/unstable.list
+deb http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
```

Comment:
As we're using Debian stretch, this should now be http://deb.debian.org/ (see that page for an explanation, and https://wiki.debian.org/DebianGeoMirror).